src/dishka/exceptions.py

```diff
@@ -81,3 +81,7 @@ def __str__(self) -> str:
 
 class GraphMissingFactoryError(NoFactoryError, InvalidGraphError):
     pass
+
+
+class NotOverrideFactoryError(DishkaError):
```

Comment:
I guess we need 2 errors:
* override=True while there is nothing to override
* override=False while it there is another factory

And find some better naming.
And, we need to print both factories locations in the error